src/app/showcase/alert-showcase/alert-showcase.component.ts

```diff
@@ -18,7 +18,8 @@ export class AlertShowcaseComponent {
     },
     {
       name: 'message',
-      description: '(Optional) The message shown under the title (or icon if specified)',
+      description:
+        "(Optional) The message shown under the title (or icon if specified). Use '\\n' for newline",
```

Comment:
Isn't it only 1 backslash?
```suggestion
        "(Optional) The message shown under the title (or icon if specified). Use '\n' for newline",
```

Reply:
the first one escapes the second one